coderedcms/models/snippet_models.py

```diff
@@ -224,6 +226,15 @@ class Meta:
         max_length=255,
         verbose_name=_('Name'),
     )
+    layoutsetting = models.ForeignKey(
```

Comment:
Would it make more sense to have a Foreign Key to a `Site` rather than a settings object?